plugins/modules/site_workflow_manager.py

```diff
@@ -731,15 +1038,36 @@ def get_have(self, config):
         current_site = None
         have = {}
 
-        # check if given site exits, if exists store current site info
+        try:
+            if self.dnac_version >= self.version_2_3_7_6 and self.config[0]['site']['type'] == "bulk":
+                (site_exists, current_site) = self.site_exists()
+                self.log("Current Site details (have): {0}".format(
+                    str(current_site)), "DEBUG")
+                if site_exists:
+                    have["site_exists"] = site_exists
+                    have["current_site"] = current_site
+                    self.log(current_site)
+                return self
+
+        except Exception as e:
+            self.log("An unexpected error occurred: {0}".format(e), "ERROR")
```

Comment:
set the status to failed and return from here if you get any exception - 
```
self.status = "failed"
self.msg = "An error occured while getting the site details due to : {0}".format(str(e))
self.log(self.msg, "ERROR")
self.result["response"] = self.msg
return self

Reply:
Added